### ISSUE TITLE
Quacker doesn't load tweets anymore[BUG]

### ISSUE THREAD

**Issue by Daniongithub** [TheHCJ/Quacker#20]
**Describe the bug**

**To Reproduce**
1. Go to 'feed/user/search result'
3. See error

**Expected behavior**
Error: Unable to load the tweets for the feed
Instance of 'qB'

**Screenshots**
![IMG_20230815_103505](https://github.com/TheHCJ/Quacker/assets/135717077/94da744e-5d73-4c99-b2bf-2f25049f84bd)

**System Infomation (please complete the following information):**
 - OS: [MIUI v.12.5.4.0]
 - Version: [v3.0.0]

**Additional context**
It all started this morning. (UTC+2)

**Comment by Daniongithub** [comment]
Oops, i didn't want to close it

**Comment by nannadroid** [comment]
Yes, I'm also seeing. Was working until few hours ago. [Same issue](https://github.com/jonjomckay/fritter/issues/768#issue-1848598100) is raised in Fritter as well.

**Comment by TheHCJ** [comment]
I am aware and I am working on a fix

**Comment by TheHCJ** [comment]
For now you can use the twitter embed:
https://syndication.twitter.com/srv/timeline-profile/screen-name/[USERNAME HERE]?showReplies=false

**Comment by Daniongithub** [comment]
> I am aware and I am working on a fix

Thanks

**Comment by TheHCJ** [comment]
Things are working again, I will be publishing a fix soon to auto refresh when rate limited

**Comment by TheHCJ** [comment]
It broke again

**Comment by Daniongithub** [comment]
What's happening on "the behind" these days? Did Twitter still limit the API?

**Comment by TheHCJ** [comment]
Yeah, you now need an account to browse X. Nitter has solved the problem by
using android guest accounts via a residential proxy. Due to this I will
have to convert Quacker to use Nitter for it's data.

**Comment by TheHCJ** [comment]
Twitter stopped guest auth so nitter has had to use android guest accounts. This could be possible with Quacker and by extension Fritter. I am looking into a solution.
